app/uk/gov/hmrc/exports/models/declaration/submissions/Submission.scala

```diff
@@ -39,8 +39,21 @@ case class Submission(
 
 object Submission {
 
-  implicit val format = Json.format[Submission]
+  implicit def format = Json.format[Submission]
```

Comment:
Why a `def`? Is it dynamic in some way?

Reply:
Maybe left over from development - will adjust now